### PR TITLE
story/EUI-5113 completed hearing

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5344

### Change description ###
View details of Hearings from Hearings tab- Adjourned hearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/hearings/data/hearing-actuals-completed.mock.data.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import { HearingResult } from '../../../src/hearings/models/hearings.enum';
```
Comment (rayliangatos) [inline api/hearings/data/hearing-actuals-completed.mock.data.ts:1]: could this refer to api one?

File: api/hearings/data/hearing-actuals-completed.mock.data.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+      },
+      {
+        categoryType: CategoryType.CaseSubType,
+        categoryValue: 'Rate of Assessment / Payability Issues - complex',
+      }],
+    caseManagementLocationCode: null,
+    caserestrictedFlag: false,
+    caseSLAStartDate: '2021-11-23T09:00:00.000+0000',
+  },
+};
```
Comment (rayliangatos) [inline api/hearings/data/hearing-actuals-completed.mock.data.ts:244]: could this put to the existing hearing-actuals.mock.data.ts?

File: api/hearings/data/hearingRequests.mock.data.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import {PartyType} from '../../../src/hearings/models/hearings.enum';
-import {HearingRequestMainModel} from '../models/hearingRequestMain.model';
-import {CategoryType, LaCaseStatus, ListingStatus} from '../models/hearings.enum';
+import { PartyType } from '../../../src/hearings/models/hearings.enum';
```
Comment (rayliangatos) [inline api/hearings/data/hearingRequests.mock.data.ts:1]: src

File: src/hearings/hearings.routes.ts [hunk 4]
```diff
@@ -123,6 +124,21 @@ export const ROUTES: Routes = [
...
+          courtLocation: CourtLocationsDataResolver,
+          partyChannels: HearingActualPartyChannelResolverService,
+        },
+        component: HearingCompletedSummaryComponent,
+        canActivate: [HealthCheckGuard],
+        data: {
+          title: 'HMCTS Hearings | Hearing Actuals | Check details',
+          isChildRequired: true
+        }
+      },
```
Comment (rayliangatos) [inline src/hearings/hearings.routes.ts:141]: could this move to the view path?

File: src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l">{{hearingActualsMainModel?.caseDetails.publicCaseName}}</span>
+      Hearing details
+    </h1>
+
+    <exui-listing-information-summary></exui-listing-information-summary>
```
Comment (rayliangatos) [inline src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.html:8]: Could we put this <exui-listing-information-summary></exui-listing-information-summary> back to src/hearings/containers/view-hearing/hearing-completed-summary/hearing-completed-summary.component.html? As basic common component shouldn't be inside another basic common component.

File: src/hearings/containers/view-hearing/hearing-completed-summary/hearing-completed-summary.component.html [hunk 6]
```diff
@@ -0,0 +1 @@
+<exui-hearing-actual-summary></exui-hearing-actual-summary>
```
Comment (rayliangatos) [inline src/hearings/containers/view-hearing/hearing-completed-summary/hearing-completed-summary.component.html:1]:
```suggestion
<div class="govuk-grid-row">
  <div class="govuk-grid-column-full">
    <h1 class="govuk-heading-l">
      <span class="govuk-caption-l">{{hearingActualsMainModel?.caseDetails.publicCaseName}}</span>
      Hearing details
    </h1>
    <exui-listing-information-summary></exui-listing-information-summary>
    <exui-hearing-actual-summary></exui-hearing-actual-summary>
  </div>
</div>
```
And build the exui-hearing-actual-summary as atomic as possible.

**Review by rayliangatos — APPROVED** [review]
Thanks, looks great to me.

**Review by SivaprakasamSivaprakasam — APPROVED** [review]

**Pull request closed without merging**